### PR TITLE
BREAKING: Bump Node to v14

### PR DESCRIPTION
**Pull request by bowensanders**
This changes all refs to v12 to point to v14 as v12 is no longer LTS

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x]
```
Comment (rekmarks) [inline .github/workflows/build-test.yml:14]: We should add `18.x` to this array as well.

**Review by rekmarks — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (bowensanders) [reply .github/workflows/build-test.yml:14]: Requesting permission to wait to add 18.x support to this PR (and the few others) until after vacation.

Comment (FrederikBolding) [reply .github/workflows/build-test.yml]: Done

**Review by Mrtenz — CHANGES_REQUESTED** [review]
Please update `README.md` too.

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**